src/poliastro/twobody/orbit.py

```diff
@@ -162,7 +168,19 @@ def from_body_ephem(cls, body, epoch=None):
             warn("Input time was converted to scale='tdb' with value "
                  "{}. Use Time(..., scale='tdb') instead."
                  .format(epoch.tdb.value), TimeScaleWarning)
+        if body == Moon:
+            solar_system_ephemeris.set("jpl")
         r, v = get_body_barycentric_posvel(body.name, epoch)
+        if body == Moon:
+            moon_icrs = ICRS(x=r.x, y=r.y, z=r.z, v_x=v.x, v_y=v.y, v_z=v.z,
+                             representation=CartesianRepresentation, differential_type=CartesianDifferential
+                             )
+            moon_gcrs = moon_icrs.transform_to(GCRS(obstime=epoch))
+            moon_gcrs.representation = CartesianRepresentation
+            return cls.from_vectors(
```

Comment:
You can avoid having two `return` statements by carefully setting the attractor, the position and velocity inside this `if` branch.

Reply:
Yes, I did that! Thanks for pointing this out!